app/assets/stylesheets/style.css

```diff
@@ -0,0 +1,39 @@
+
+html, body {
```

Comment:
such global styles like css for html and body we prefer to keep in /assets/stylesheets/core in one of that files. For example 'base.scss' is good place to keep site-wide styles